### ISSUE TITLE
no more print settings' preferences in print view

### ISSUE THREAD

**Issue by TurboGit** [darktable-org/darktable#10892]
**Describe the bug/issue**

The print settings' preferences (hamburger) is not active anymore. This means that it is not possible to use a presets or create a new one.

Note that this is a regression and also reproducible in future 3.8.1.

**To Reproduce**
1. Go to 'Print' view
2. Click on 'preferences' of print settings lib
3. See error, not active

**Which commit introduced the error**

```
6fbfeb1b7183449fb6aa30dee13ffab1d3e40aa9 is the first bad commit
commit 6fbfeb1b7183449fb6aa30dee13ffab1d3e40aa9
Author: Diederik ter Rahe <dterrahe@yahoo.com>
Date:   Sat Jan 1 15:19:26 2022 -0500

    register lib presets before loading shortcuts

 src/libs/lib.c            | 36 +++++++++++++++---------------------
 src/libs/lib.h            |  3 ---
 src/libs/location.c       |  3 ---
 src/libs/metadata_view.c  |  4 ----
 src/libs/print_settings.c |  4 ----
 src/views/view.c          |  1 -
 6 files changed, 15 insertions(+), 36 deletions(-)
```

(also merged in darktable-3.8.x branch)

**Platform**

* darktable version : 3.9
* OS                : Linux 
* Linux - Distro    : GNU/Debian - sid

**Comment by TurboGit** [comment]
@dterrahe : Please see #10893. A simpler fix for 3.8.1 could be to add back the init_presets() into print_settings.c but I wanted to get this clean.

**Comment by dterrahe** [comment]
Ah, sorry, didn't notice your PR and updates here while I was looking at the code trying to figure out what was going on. A lot of implicit dependencies.
